function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
```

Comment:
Function has no name

Reply:
I agree this function needs to have a name, just as a reminder functions have: 
 - name > the function name.
- param > the name of an argument to be passed to the function. Maximum number of arguments varies in different 
  engines. 
- statements> the statements which comprise the body of the function